src/scss/custom/_index.scss

```diff
@@ -0,0 +1,9 @@
+@import '../../../node_modules/bootstrap/scss/functions';
+
+.clientAplication {
```

Comment:
these classes should be in a separate file that you import into index from the custom folder.

Reply:
I will fix this when its merged